scripts/generate-api.cjs

```diff
@@ -6,6 +6,14 @@ const SCHEMA_MANAGER_URL = 'https://affinidi-schema-manager.prod.affinity-projec
 const IAM_URL = 'https://affinidi-iam.prod.affinity-project.org'
 const ISSUANCE_URL = 'https://console-vc-issuance.prod.affinity-project.org'
 const VERIFY_URL = 'https://affinity-verifier.prod.affinity-project.org'
+const ANALYTICS_URL = 'https://analytics-stream.staging.affinity-project.org'
```

Comment:
Shouldn't we use the prod URL here as well?

Reply:
the prod api is hidden, this is just to generate http client but the events are sent to prod